miri/datamodels/ancestry.py

```diff
@@ -62,6 +65,7 @@
                'MiriLinearityModel' : 'LinearityModel', \
 #                'MiriPceModel' : 'PhotomModel', \
                'MiriImagingPhotometricModel' : 'MirImgPhotomModel', \
+               'MiriLrsPhotometricModel' : 'MirLrsPhotomModel', \
```

Comment:
For future reference, this maps to the MirLrsPhotomModel defined at:
https://github.com/spacetelescope/jwst/blob/74ed6b27b0537135c2dab1a989f412121f19c3e2/jwst/datamodels/photom.py#L52